sonic_api.yaml

```diff
@@ -161,6 +162,8 @@ paths:
 #----------------------------------------------
 # Config Reset Status API
 #----------------------------------------------
+# schema_violation | Schema violation: must NOT have additional properties (paths > /config/resetstatus > post > parameters > 0)
+# schema_violation | Schema violation: must have required property 'schema' (paths > /config/resetstatus > post > parameters > 0)
```

Comment:
Is the API for post here expecting a true/false value as the only item in post? or is it expecting a json blob or something?

Reply:
We expect a {"reset_status": "true/false"} in POST params

Reply:
ok, will validate the generated client for this does actually flatten like that

Reply:
Generated client did not generate with a json blob as the payload, so I've added a schema which does generate properly